koku/masu/database/gcp_report_db_accessor.py

```diff
@@ -307,8 +307,14 @@ def get_gcp_topology_trino(self, source_uuid, start_date, end_date, invoice_mont
                 service_description,
                 location_region
         """
-
-        return self._execute_trino_raw_sql_query(sql, log_ref="get_gcp_topology_trino")
+        sql_params = {
+            "schema": self.schema,
+            "start": start_date,
+            "end": end_date,
+            "provider_uuid": source_uuid,
+            "invoice_id": invoice_month_date,
+        }
```

Comment:
Adding these sql_params strictly for logging makes this code harder to understand because it _seems_ like these should be substituted into the sql (but they are not). I think it would make more sense for us to make these actual jinja templates too just so that this code and the use of these params makes sense. This template could be:
```
        sql = """
            SELECT source,
                billing_account_id,
                project_id,
                project_name,
                service_id,
                service_description,
                location_region
            FROM hive.{{schema | sqlsafe}}.gcp_line_items as gcp
            WHERE gcp.source = '{{source_uuid}}'
                AND gcp.year = '{{year}}'
                AND gcp.month = '{{month}}'
                AND gcp.usage_start_time >= {{start_date}}
                AND gcp.usage_start_time < date_add('day', 1, {{end_date}})
            GROUP BY source,
                billing_account_id,
                project_id,
                project_name,
                service_id,
                service_description,
                location_region
        """
        sql_params = {
            "schema": self.schema,
            "start_date": start_date,
            "end_date": end_date,
            "source_uuid": source_uuid,
            "year": invoice_month_date.strftime("%Y"),
            "month": {invoice_month_date.strftime("%m")},
            "invoice_id": invoice_month_date,
        }
```

That, OR we add a `context` param to the `_execute_trino_raw_sql_query` just so that it is clear that those params are passed strictly for logging and not to be used as sql_params.

Reply:
Yeh your not wrong! I was just trying to force them into the logging dirty like (without changing everything) just to see if it helped follow things a little better for Ford. :smile:

Reply:
Context it is :+1: